Story/OnlineUIComponents/PlayerSpecificOnlineHud.cs

```diff
@@ -123,20 +123,19 @@ public override void Update()
                 }
             }
 
-            if (!clientSettings.inGame) return;
+            this.found = false;
+
             if (camera.room == null || !camera.room.shortCutsReady) return;
-            if (abstractPlayer == null)
+            if (!clientSettings.inGame) return;
+            if (clientSettings.avatars.Count == 0) return;
+            if (clientSettings.avatars[0]?.FindEntity(true) is OnlineCreature oc) // TODO: support multiple avatars
```

Comment:
I don't like this but
1. negligible performance cost
2. not enough spoons to refactor everything